### ISSUE TITLE
[Bukkit Forge] Tekkit Lite 0.6.1 BUILD # 257

### ISSUE THREAD

**Issue by enterprise12nx** [keepcalm/BukkitForge#581]
2013-03-18 15:14:28 [INFO] [STDOUT] THIS SERVER IS RUNNING BUKKITFORGE null-null. JUST IN CASE SOMEONE ASKS!
2013-03-18 15:14:29 [INFO] [STDERR] Exception in thread "main" java.lang.NoClassDefFoundError: gm
2013-03-18 15:14:29 [INFO] [STDERR]     at java.lang.Class.getDeclaredMethods0(Native Method)
2013-03-18 15:14:29 [INFO] [STDERR]     at java.lang.Class.privateGetDeclaredMethods(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at java.lang.Class.getDeclaredMethod(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.ReflectionHelper.findMethod(ReflectionHelper.java:170)
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.FMLRelauncher.relaunchServer(FMLRelauncher.java:138)
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.FMLRelauncher.handleServerRelaunch(FMLRelauncher.java:33)
2013-03-18 15:14:29 [INFO] [STDERR]     at net.minecraft.server.MinecraftServer.main(MinecraftServer.java:1376)
2013-03-18 15:14:29 [INFO] [STDERR] Caused by: java.lang.ClassNotFoundException: gm
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.RelaunchClassLoader.findClass(RelaunchClassLoader.java:185)
2013-03-18 15:14:29 [INFO] [STDERR]     at java.lang.ClassLoader.loadClass(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at java.lang.ClassLoader.loadClass(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     ... 7 more
2013-03-18 15:14:29 [INFO] [STDERR] Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: ig
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.ClassWriter.getCommonSuperClass(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.ClassWriter.a(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.Frame.a(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.Frame.a(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.MethodWriter.visitMaxs(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.tree.MethodNode.accept(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.tree.MethodNode.accept(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at org.objectweb.asm.tree.ClassNode.accept(Unknown Source)
2013-03-18 15:14:29 [INFO] [STDERR]     at com.eoware.asm.asmagic.AsmagicClassTransformer.transformClass(AsmagicClassTransformer.java:89)
2013-03-18 15:14:29 [INFO] [STDERR]     at com.eoware.asm.asmagic.AsmagicClassTransformer.transform(AsmagicClassTransformer.java:30)
2013-03-18 15:14:29 [INFO] [STDERR]     at keepcalm.mods.bukkit.asm.transformers.BukkitAsmagicTransformer.transform(BukkitAsmagicTransformer.java:67)
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.RelaunchClassLoader.runTransformers(RelaunchClassLoader.java:228)
2013-03-18 15:14:29 [INFO] [STDERR]     at cpw.mods.fml.relauncher.RelaunchClassLoader.findClass(RelaunchClassLoader.java:173)
2013-03-18 15:14:29 [INFO] [STDERR]     ... 9 more

Also now it says my world is leaking :(

**Comment by sim242** [comment]
Same problem

**Comment by enterprise12nx** [comment]
It seems Tekkit Lite 0.6.1 is completely broken, the client and server side are incompatible mods because of modular power suits,
ChickenChunks mod is completely screwed up, it is making world leaks, so i believe they have to remove that.
IT may be some time before any of this is resolved, meaning no server will be working unless using 0.5.7

**Comment by sim242** [comment]
Found the fix mate. Run Bukkit Forge build 250. Yeah it's a few builds old but it works. :) Your welcome

**Comment by enterprise12nx** [comment]
There is still the problem with the world leaks with version 0.6.1 with ChickenChunks Mod
i cant connect to www.technicpack.net with the launcher either because they are either flooded with traffic or doing fixes on the pack

**Comment by SpoonsJTD** [comment]
No idea what you are talking about. Are you trying to run MC 1.5?

**Comment by enterprise12nx** [comment]
Im running tekkit lite, and i updated to 0.6.1 and then i started getting 

2013-03-18 15:42:39 [FINE] [DimensionalAnchors] Requesting chunk loading ticket for the overworld
2013-03-18 15:42:39 [INFO] [ForgeModLoader] Unloading dimension 1
2013-03-18 15:42:39 [INFO] [ForgeModLoader] Unloading dimension -23
2013-03-18 15:42:49 [SEVERE] [ForgeModLoader] Detected leaking worlds in memory. There are 4 worlds that appear to be persisting. A mod is likely caching the world incorrectly

2013-03-18 15:42:49 [SEVERE] [ForgeModLoader] The world 3e9e08b9 (world) has leaked.

2013-03-18 15:42:49 [SEVERE] [ForgeModLoader] The world 1c7057ef (world) has leaked.

I even tryed a fresh install of 0.6.1 flat, still persists with or without bukkitforge and i hear its the ChickenChunk mod.  its a chunk loading mod, also the client is only in 0.5.7, there is no build for 0.6.1 even latest, 
i also hear that the client and server arent compatible right now (which i have proved when ppl tryed to join the server)
the ModularPowerSuits Mods are different versions, they need to fix it so people can join.

they also have no changelog for 0.6.1

**Comment by Erudition** [comment]
Same problem!
So no 0.6.1 then?
Or just no bukkitforge 257?

**Comment by enterprise12nx** [comment]
No 0.6.1
The tekkit authors have to fix it, cause all servers will have a server leak, which i believe is a memory RAM leak, uses a crap ton of RAM not sure, but not good.
i would say it could be both sides, but just alone its Tekkit Lites Problem.
All we can do is wait

**Comment by dequis** [comment]
Seriously guys. SERIOUSLY. Search the bug tracker before adding new bugs, and if you do that, you can find some workarounds: #570

tl;dr either remove treecapitator or downgrade bukkitforge to 252.

@SpoonsJTD close this duplicate

edit: the tekkit lite memory leak isn't relevant here at all...

**Comment by sim242** [comment]
I can't work out world leaks sorry

**Comment by enterprise12nx** [comment]
Im aware its not your problem, but we were adressing that there is a possible problem with the version, and i tryed to help both problems while at it.

**Comment by SpoonsJTD** [comment]
Dupe

**Comment by Ionaru** [comment]
TekkitLite has the mod Treecapitator, which does not work with any bukkitforge version above 252. Either remove the mod or downgrade to that bukkitforge version.

**Comment by enterprise12nx** [comment]
oh..why D:
that means i cant update it..cause if i remove the mod nobody could join, why does it not have support /=

**Comment by Erudition** [comment]
if you remove the mod, everyone will join just fine.
try it.

**Comment by sim242** [comment]
Does removing that mod prevent world leaks?
